net/modemmanager/files/modemmanager.proto

```diff
@@ -342,6 +344,21 @@ proto_modemmanager_setup() {
 		return 1
 	}
 
+	echo "Waiting for Modem initialization"
```

Comment:
Are we truly sure this is right? If netifd attempts to setup the interface and ModemManager is not yet running. the calls below grabbing modemstatus and modempath will end up failing, and the `DEVICE_NOT_MANAGED` error will happen. 

Then, as soon as MM starts, the hotplug scripts will notify that the interface is up and netifd will set it again.

Reply:
Will also comment in the original bugreport